extensions/wikia/SpecialHealthcheck/HealthController.class.php

```diff
@@ -165,6 +162,34 @@ private function testHost( $databaseName, LoadBalancer $loadBalancer, $index ) {
 			return false;
 		}
 
+		return true;
+	}
+
+	/**
+	 * Execute checks for a single database server
+	 * @param LoadBalancer $loadBalancer Load Balancer instance for the given cluster
+	 * @param int $index Server index to test
+	 * @return bool Is server healthy?
+	 * @throws MWException
+	 */
+	private function testHost(LoadBalancer $loadBalancer, $index ) {
+		// connection check
+		try {
+			$db = wfGetDB( $index, array() );
```

Comment:
Will this DB connection always be for Community Central since that's the wiki this is being requested on and it would use that loadbalancer rather than the one for the DB cluster we want to check?